Improve example of .flex-wrap and -reverse

stollr opened the pull request:
### Description

Added numbers to the labels in the example of `.flex-wrap` and `.flex-wrap-reverse` to make it more clear what happens and how `.flex-wrap-reverse` works.

### Motivation & Context

The current example of `.flex-wrap` and `.flex-wrap-reverse` does not really show the difference between those two utilities, because every item is labelled 'Flex item'.

### Type of changes

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would change existing functionality)

### Checklist

<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [x] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [ ] My code follows the code style of the project _(using `npm run lint`)_
- [x] My change introduces changes to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### Notice

If you like this PR I'd be happy if you would merge it or label it with [`hacktoberfest-accepted`](https://hacktoberfest.com/participation/). Thanks in advance.

### [Live preview](https://deploy-preview-37279--twbs-bootstrap.netlify.app/docs/5.2/utilities/flex/#wrap)

julien-deramond reviewed (changes requested):
It might be useful to add those numbers to understand what happens with `.flex-wrap-reverse`. @mdo thoughts?

Commit:
Improve example of .flex-wrap and -reverse

The current example of `.flex-wrap` and `.flex-wrap-reverse` does not really show the difference between those two utilities, because every item is labelled 'Flex item'.

With the added numbers it is more clear what happens and how `.flex-wrap-reverse` works.

naitsirch commented on site/content/docs/5.2/utilities/flex.md:
```diff
@@ -363,21 +363,20 @@ Change how flex items wrap in a flex container. Choose from no wrapping at all (
-    <div class="p-2">Flex item</div>
-    <div class="p-2">Flex item</div>
-    <div class="p-2">Flex item</div>
+    <div class="p-2">Flex item1</div>
```
I have updated the PR according to your suggestion ;-)
